htdocs/helpFiles/Entering-Units.html

```diff
@@ -191,11 +212,11 @@ <h2 style="text-align: center">Units Available in WeBWorK</h2>
 	</tr>
 	<tr>
 		<td style="padding: 0.5rem; border: 1px solid black">Angle degrees</td>
-		<td style="padding: 0.5rem; border: 1px solid black; text-align: center">deg</td>
+		<td style="padding: 0.5rem; border: 1px solid black; text-align: center">deg, º, degree or degrees</td>
```

Comment:
Should the special character be raw like this in source? I think we want it to be `&#x00B0;` here.

Reply:
Will switch.